docs/live-sim.mdx

```diff
@@ -0,0 +1,98 @@
+
+:::
+
+## Simulate your machine with others
+
+1. Select the **Simulate** mode from the center of the visual editor. When in the **Simulate** mode, you’ll see your machine with the initial state turn the color blue. Blue means your machine is not currently being edited or shared, but you are simulating the logic flow of the current machine. In **Simulate** mode, this is where you can go through each state and transition to thoroughly review your logic.
+
+<p>
+<ThemedImage
+  alt="Simulate mode is selected in the center menu and now the machine can not be edited."
+  sources={{
+    light: useBaseUrl('/live-sim/live-sim-menu.png'),
+    dark: useBaseUrl('/live-sim/live-sim-menu-dm.png'),
+  }}
+/>
+</p>
+
+2. While in **Simulate** mode, you’ll notice a button **Live mode** on the bottom left corner. When clicking the **Live mode** button, you’ll have the options: 
+- **Join a Live simulation** session
+- **Start a live simulation** session.
+
+:::tip
+
+The **Live mode** button will only be displayed when you are using the Simulate mode in the Studio. To know if you’re in the Simulate mode, you’ll see your states and transitions light up in blue color.
+
+:::
+
+3. When selecting the option **Start a live simulation**, you’ll notice the machine is orange. Then, the bottom left corner will show **Copy link** or **Stop** buttons, along with a 6 character code (for example: 3KNYII). 
+You can share your machine when in Live simulation mode by selecting the option **copy link** and then a modal will pop up to confirm the link was copied to your clipboard. Go ahead and share that link to other participants! 
+The link can also be accessed by users who are not logged in to the Studio.
+
+<p>(TODO:add screenshot here)
+<ThemedImage
+  alt="When selecting Start a live simulation, you are actively sharing the machine and others can access the same simulation with a custom generated link."
+  sources={{
+    light: useBaseUrl('/live-sim/live-sim-start.png'),
+    dark: useBaseUrl('/live-sim/live-sim-start-dm.png'),
+  }}
+/>
+</p>
+
+::: tip
+
+To recognize you’re in a Live simulation mode, you’ll notice the blue states have now transitioned to orange.
+
+:::
+
+4. When selecting the option **Join a Live simulation**, you’ll be prompted to enter a code to reference which machine simulation session you’ll be joining.
+If you were given a link to join the Live simulation session, you would only need to copy and paste that link into your browser.
+The two buttons that pop up are to **Cancel** or **Join**, when you have joined a Live simulation session. 
+
+<p>
+<ThemedImage
+  alt="Joining a Live simulation session will all you to select the states or watch them transition to other states with others in the session."
+  sources={{
+    light: useBaseUrl('/live-sim/live-sim-join.png'),
+    dark: useBaseUrl('/live-sim/live-sim-join-dm.png'),
+  }}
+/>
+</p>
+
+### Benefits
```

Comment:
I reckon we keep this whole benefits section for the blog post. It’s really great stuff, but may not be necessary for knowing how to use the feature itself. What do you think?